ratelimit/migrations/0001_initial.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.2 on 2020-11-03 11:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ExceededLimitRecord',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
> This probably needs a bit of rework with the conflicts that have cropped up from some other refactors.
> 
> One thing I want to avoid is baking too much optional behavior into the "core." I wonder if we could tuck some of this into something like `ratelimit.ext.recorder` with its own `AppConfig`, so that installation requires explicit opt-in like `INSTALLED_APPS += ['ratelimit.ext.recorder']` and we don't end up installing unused models or causing inadvertent migrations.

I have updated the PR and fixed the conflicts. Sadly it's hard for me to find some spare time to work on this more but, I will make some more adjustments. I appreciate any help 👍